static/app/views/issueDetails/streamline/eventMissingBanner.tsx

```diff
@@ -29,7 +32,10 @@ export function EventMissingBanner() {
       'Switch over to a [link:recommended] event instead, it should have more useful data.',
       {
         link: (
-          <Link to={`${baseUrl}/recommended/`} aria-label={t('View recommended event')} />
+          <Link
+            to={`${baseUrl}/recommended/statsPeriod=${MAX_PICKABLE_DAYS}d`}
```

Comment:
i think this is formatted wrong and i think we'd want to preserve the project id here too
```suggestion
            to={`${baseUrl}/recommended/?statsPeriod=${MAX_PICKABLE_DAYS}d`}
```

Reply:
Oops Cursor led me astray mb, this is completely wrong :+1: